### PR TITLE
Added colors to BP Controlled, BP Uncontrolled, and Missed Visits %s

### PR DESCRIPTION
**Pull request by dburka**
Added colors to BP Controlled, BP Uncontrolled, and Missed Visits percentages to make the tables easier to scan.

**Story card:** None

## Because

When all of the percentages are purple, it's hard to scan tables to see who is performing well and who is not.

## Test instructions

Shouldn't break anything outside of the tables it's meant to affect. Click around My Facilities pages, click around Reports tables, and formatting of % pills should be colored appropriately instead of purple.

Note, the values are relatively arbitrary based on how we think IHCI is valuing performance. It's rough.

### CODE REVIEW

File: app/views/my_facilities/bp_controlled.html.erb [hunk 1]
```diff
@@ -168,6 +168,30 @@
             </tbody>
           </table>
         <% end %>
+        <div class="d-flex mt-4">
+          <em class="h-12px w-12px mr-8px bg-red br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            0-20%
+          </p>
+        </div>
```
Comment (claudiovallejo) [inline app/views/my_facilities/bp_controlled.html.erb:176]: There should be a separate grey key for `0%` as grey to match the `.high-is-good` CSS.

File: app/views/my_facilities/bp_controlled.html.erb [hunk 2]
```diff
@@ -168,6 +168,30 @@
...
+          <p class="fs-12px lh-1 c-grey-dark">
+            37-50%
+          </p>
+        </div>
+        <div class="d-flex">
+          <em class="h-12px w-12px mr-8px bg-green-medium br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            &gt;50%
+          </p>
+        </div>
```
Comment (claudiovallejo) [inline app/views/my_facilities/bp_controlled.html.erb:194]: The `.high-is-good` key doesn't match the CSS in `_dashboard.css`.
* `0%` is grey in CSS, but isn't listed as a key item
* `1-20%` should be red
* `21-35%` should be orange
* `36-49%` should be yellow
* `≥50%` should be green

File: app/views/my_facilities/bp_not_controlled.html.erb [hunk 3]
```diff
@@ -167,6 +167,30 @@
...
+          <p class="fs-12px lh-1 c-grey-dark">
+            11-20%
+          </p>
+        </div>
+        <div class="d-flex">
+          <em class="h-12px w-12px mr-8px b-2px bg-green-medium br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            0-10%
+          </p>
+        </div>
```
Comment (claudiovallejo) [inline app/views/my_facilities/bp_not_controlled.html.erb:193]: The `BP not controlled` and `Missed visits` pill key doesn't match the CSS in `_dashboard.css`.
❌ `0%` is grey in CSS, but isn't listed as a key item
❌ `1-10%` should be green
✔️ `11-20%` should be yellow
✔️ `21-30%` should be orange
❌ `≥31%` should be red

File: app/views/my_facilities/missed_visits.html.erb [hunk 4]
```diff
@@ -167,6 +167,30 @@
...
+          <p class="fs-12px lh-1 c-grey-dark">
+            11-20%
+          </p>
+        </div>
+        <div class="d-flex">
+          <em class="h-12px w-12px mr-8px bg-green-medium br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            0-10%
+          </p>
+        </div>
```
Comment (claudiovallejo) [inline app/views/my_facilities/missed_visits.html.erb:193]: Look at `BP not controlled` comment: https://github.com/simpledotorg/simple-server/pull/3624/files#r831377196

File: app/views/my_facilities/bp_controlled.html.erb (hunk 2 above)
Comment (dburka) [reply app/views/my_facilities/bp_controlled.html.erb:194]: Done. Good catches.

File: app/views/my_facilities/bp_not_controlled.html.erb (hunk 3 above)
Comment (dburka) [reply app/views/my_facilities/bp_not_controlled.html.erb:193]: Done

File: app/views/my_facilities/missed_visits.html.erb (hunk 4 above)
Comment (dburka) [reply app/views/my_facilities/missed_visits.html.erb:193]: Done

File: app/views/reports/regions/_child_data_tables.html.erb [hunk 5]
```diff
@@ -183,7 +183,7 @@
             data-toggle="tooltip"
             title="<%= number_with_delimiter(result.dig(:missed_visits, @period)) %> / <%= number_with_delimiter(result.dig(:adjusted_patient_counts, @period)) %> patients"
           >
-            <em data-rate="<%= result.dig(:missed_visits_rate, @period) %>">
+            <em data-rate="<%= result.dig(:missed_visits_rate, @period) %>" class="low-is-good">
```
[the remark this answers is not in the document]
Comment (dburka) [reply app/views/reports/regions/_child_data_tables.html.erb:186]: Done

**Pull request merged**